bazel/BUILD

```diff
@@ -987,3 +948,28 @@ updater(
     version_file = "@envoy_api//bazel:repository_locations.bzl",
     version_path_replace = "external/envoy_api:api",
 )
+
+cc_library(
+    name = "tcmalloc_libs",
```

Comment:
This is probably going to be somewhat confused with `tcmalloc_lib`.
It is worth considering either renaming (e.g., `tcmalloc_all_libs`) or removing the single-lib version if it is not really needed.

Reply:
both are needed - for slightly different reasons - ill rename